src/oge/consumed.py

```diff
@@ -32,6 +32,7 @@
     2021: ["CPLW", "GCPD"],
     2022: ["CPLW", "GCPD", "HST"],
     2023: ["CPLW"],
+    2024: ["CPLW", "SEC", "SWPP"],
```

Comment:
I'm surprised that SWPP is on this list. I'm wondering if there's some additional data cleaning we need to do.